monkey/monkey_island/cc/services/utils/node_states_test.py

```diff
@@ -0,0 +1,21 @@
+from unittest import TestCase
+
+from monkey_island.cc.services.utils.node_states import NodeStates, NoGroupsFoundException
+
+
+class TestNodeGroups(TestCase):
+
+    def test_get_group_by_keywords(self):
```

Comment:
Use `self.assertEqual(NodeStates.get_by_keywords(['island']), NodeStates.ISLAND)` instead of assertTrue for tests 1,2 and 3
Use `self.assertRaises(ExpectedException, afunction)` for test 4